packages/core-flows/src/fulfillment/workflows/delete-shipping-options.ts

```diff
@@ -0,0 +1,16 @@
+import { FulfillmentWorkflow } from "@medusajs/types"
+import { createWorkflow, WorkflowData } from "@medusajs/workflows-sdk"
+import { deleteShippingOptionsStep } from "../steps"
+
+export const deleteShippingOptionsWorkflowId =
+  "delete-shipping-options-workflow"
+export const deleteShippingOptionsWorkflow = createWorkflow(
+  deleteShippingOptionsWorkflowId,
+  (
+    input: WorkflowData<FulfillmentWorkflow.DeleteShippingOptionsWorkflowInput>
+  ): WorkflowData => {
+    const serviceZones = deleteShippingOptionsStep(input.ids)
```

Comment:
this is linkable no?
we can use `removeRemoteLinkStep` after that

Reply:
💪